.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Ruby CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby-version:
+          - 3.0.0
+          - 2.7.2
+          - 2.6.6
+          - 2.5.8
+        experimental: [false]
+        include:
+          - ruby-version: head
+            experimental: true
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+      - name: Install dependencies
+        run: |
+          bundle config set --local path '../../vendor/bundle'
+          bundle install
```

Comment:
Unneeded, and most likely breaks bundler caching, see https://github.com/ruby/setup-ruby/#caching-bundle-install-automatically